Nessie-iOS-Wrapper/Account.swift

```diff
@@ -114,9 +119,13 @@ open class AccountRequest {
             if (error != nil) {
                 completion(nil, error)
             } else {
-                let json = JSON(data: data!)
-                let response = BaseResponse<Account>(data: json)
-                completion(response.object, nil)
+                do {
+                    let json = try JSON(data: data!)
```

Comment:
Same here and for all occurrences please.

Reply:
Look at comment above please